ui/store/actions.ts

```diff
@@ -2638,6 +2638,16 @@ export function hideLoadingIndication(): Action {
   };
 }
 
+/**
+ * An action creator for display a warning to the user in various places in the
+ * UI. It will not be cleared until a new warning replaces it or `hideWarning`
+ * is called.
+ *
+ * @deprecated This way of displaying a warning is confusing for users and
+ * should no longer be used.
+ * @param {string} text - The warning text to show.
```

Comment:
Ah sorry, just noticed that the type was left in. Since this file is now TypeScript, we don't need to include any types in the comments. Also this type isn't accurate anymore anyway.

```suggestion
 * @param text - The warning text to show.
```